javascript/javascript1/week1/Ez-Namey.js

```diff
@@ -0,0 +1,16 @@
+let firstWords = [ "Easy" , "Awesome" , "Corporate" ,"Cool" , "Red" ,"Bad" ,"fantactic" , "Strong" , "Cute" , "successful"]
+
+
+
+let secondWords = [ "Weak" , " Harible" ,"Perfect" , "School" , "Class" , "Classmate" ,"Roaud" , " Near" , "Street" , "Cooker"]
+
+const randomNumber = Math.floor(Math.random() * 10);
+
+
+let startupName = firstWords [randomNumber] + secondWords [randomNumber];
+
+
+//console.log(startupName)
```

Comment:
You have done a great job finding out the name for a startup :heart_eyes: 
What if we want to find a random number which is not the same for both firstWords and secondWords?

Reply:
:)

Reply:
let startupName = firstWords  + secondWords [randomNumber];